src/PHPUnit/Constraint/Crawler.php

```diff
@@ -26,14 +36,18 @@ protected function matches($nodes) : bool
         return false;
     }
 
-    protected function fail($nodes, $selector, ComparisonFailure $comparisonFailure = null):void
+    /**
+     * @param SymfonyDomCrawler $nodes
+     * @param string|array|WebDriverBy $selector
```

Comment:
Based on usage below, $selector can only be string.
This constraint isn't used by WebDriver module.